test/features/payments/payment_details_page_test.dart

```diff
@@ -12,7 +12,14 @@ void main() {
         (tester) async {
       await tester.pumpWidget(
         WidgetHelpers.testableWidget(
-          child: const PaymentDetailsPage(),
+          child: const PaymentDetailsPage(
+            inputAmount: '1.00',
+            inputCurrency: 'USD',
+            exchangeRate: '17',
+            outputAmount: '17.00',
+            outputCurrency: 'MXN',
+            transactionType: 'Deposit',
+          ),
         ),
```

Comment:
You can create a helper func for this to avoid this duplication and DRY things up a bit.

```dart
Widget _testableWidget({List<Overrid> overrides = []}) =>
  WidgetHelpers.testableWidget(
          child: const PaymentDetailsPage(
            inputAmount: '1.00',
            inputCurrency: 'USD',
            exchangeRate: '17',
            outputAmount: '17.00',
            outputCurrency: 'MXN',
            transactionType: 'Deposit',
          ),
          overrides: overrides,
        );
```

And if you want to change a value for one of these to test, you can pass that value in to the helper.